compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -339,8 +339,9 @@ setConvolutionVectorDistributionConfig(IREE::GPU::TargetAttr target,
     return failure();
   }
 
-  std::array<int64_t, 3> workgroupSize{
-      schedule->nWarpCount * targetSubgroupSize, schedule->mWarpCount, 1};
+  std::array<int64_t, 3> workgroupSize{schedule->nSubgroupCounts[0] *
+                                           targetSubgroupSize,
+                                       schedule->mSubgroupCounts[0], 1};
```

Comment:
Is it not possible for us to use the same flat workgroup size like TileAndFuse above?

Reply:
I don't know the VectorDistribute pipeline well enough to know if that is possible.